client/js/components/user/CustomerSettings/CustomerSettingsSupport.vue

```diff
@@ -197,7 +197,14 @@ export default {
     },
 
     resetForm () {
-      this.customerContact = emptyCustomer
+      const {title, phoneNumber, eMailAddress, text, visible } = emptyCustomer
```

Comment:
```suggestion
      const { title, phoneNumber, eMailAddress, text, visible } = emptyCustomer
```

Reply:
this was removed in favor of your suggestion in the comment below.